src/main/resources/de/tracetronic/jenkins/plugins/ecutest/report/atx/ATXPublisher/help.html

```diff
@@ -34,6 +37,10 @@
 def server = ATX.newServer('TEST-GUIDE', 'ECU-TEST')
 server.publish()
             </pre>
+            <pre>
+def server = ATX.newServer atxName: 'TEST-GUIDE', toolName: 'ECU-TEST'
+server.publish allowMissing: true, runOnFailed: true
```

Comment:
Other parameters like:   `archiving` and `keepAll` are not documented in this example.